# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471372262 25200
#      Tue Aug 16 11:31:02 2016 -0700
# Node ID 567c9d9228d47cc474786a4928b8cf9b6bf235b6
# Parent  a1840b139c377654c15edebfe38080f45e62ba49
isocaps.m: use size (..., 1) instead of rows in %!demos.

* isocaps.m: use size (..., 1) instead of rows in %!demos.

diff --git a/scripts/plot/draw/isocaps.m b/scripts/plot/draw/isocaps.m
--- a/scripts/plot/draw/isocaps.m
+++ b/scripts/plot/draw/isocaps.m
@@ -376,25 +376,25 @@ endfunction
 %! fvc_iso = isosurface (x, y, z, v, isoval);
 %! cmap = get (hf, "Colormap");
 %! p_iso = patch (fvc_iso, "FaceLighting", "gouraud", ...
 %!                "FaceColor", cmap(end,:), "EdgeColor", "none");
 %! isonormals (x, y, z, v, p_iso);
 %! fvc_xmin = isocaps (x, y, z, v, isoval, "xmin", "b");
 %! patch (fvc_xmin, "FaceColor", "interp", "EdgeColor", "none", ...
 %!        "FaceLighting", "gouraud", ...
-%!        "VertexNormals", repmat([-1 0 0], rows (fvc_xmin.vertices), 1));
+%!        "VertexNormals", repmat([-1 0 0], size (fvc_xmin.vertices, 1), 1));
 %! fvc_ymin = isocaps (x, y, z, v, isoval, "ymin", "b");
 %! patch (fvc_ymin, "FaceColor", "interp", "EdgeColor", "none", ...
 %!        "FaceLighting", "gouraud", ...
-%!        "VertexNormals", repmat([0 -1 0], rows (fvc_ymin.vertices), 1));
+%!        "VertexNormals", repmat([0 -1 0], size (fvc_ymin.vertices, 1), 1));
 %! fvc_zmax = isocaps (x, y, z, v, isoval, "zmax", "b");
 %! patch (fvc_zmax, "FaceColor", "interp", "EdgeColor", "none", ...
 %!        "FaceLighting", "gouraud", ...
-%!        "VertexNormals", repmat([0 -1 0], rows (fvc_zmax.vertices), 1));
+%!        "VertexNormals", repmat([0 -1 0], size (fvc_zmax.vertices, 1), 1));
 %! axis equal;
 %! light ();
 %! title ({"isocaps()", "sphere with 6 end-caps"});
 
 %!demo
 %! v = smooth3 (rand (6, 8, 4));
 %! isoval = .5;
 %! x = 1:3:22;  y = -14:5:11;  z = linspace (16, 18, 4);
